### PR TITLE
Deploy gauge adder coordinator

### PR DESCRIPTION
**Pull request by nventuro**

### CODE REVIEW

File: pkg/deployments/action-ids/mainnet/action-ids.json [hunk 1]
```diff
@@ -664,5 +664,14 @@
...
+  },
+  "20220721-gauge-adder-migration-coordinator": {
+    "GaugeAdderMigrationCoordinator": {
+      "useAdaptor": false,
+      "actionIds": {
+        "performNextStage()": "0x945b1637627fa36f3bb618b74050045ffa67875ffd8acd04f5135f3463ab9880",
+        "registerStages()": "0x08900d9a13d156ace75a5694772406e876e8f8f25d35af7a7639c1a95a2c2689"
+      }
+    }
   }
```
Comment (TomAFrench) [inline pkg/deployments/action-ids/mainnet/action-ids.json:676]: Should we just omit all the coordinators from here? I don't see any real benefit to having them included.

**Review by TomAFrench — APPROVED** [review]
Looks good.

File: pkg/deployments/action-ids/mainnet/action-ids.json (hunk 1 above)
Comment (nventuro) [reply pkg/deployments/action-ids/mainnet/action-ids.json]: Eh that's a good point. I just run the task and didn't think about it.

Comment (nventuro) [reply pkg/deployments/action-ids/mainnet/action-ids.json]: Tbh a better solution would be to have non `getActionId` contracts (i.e. those that go through the authorizer) only be added to the json if they're in some list of pre-approved contracts (gauges, etc.), as the vast majority are false positives.

**Pull request merged**